man/vma_dump_fd_stats.3.in

```diff
@@ -0,0 +1,31 @@
+.\" -*- nroff -*-
+.\"
+.TH VMA_DUMP_FD_STATS 3 "#BUILD_DATE#" libvma "VMA(#BUILD_VERSION#) User's Manual"
+.SH NAME
+vma_dump_fd_stats
+
+.SH SYNOPSIS
+.B #include <mellanox/vma_extra.h>
+.sp
+.BI "int vma_dump_fd_stats(int " "fd" ", int " "log_level" );
+
+.SH DESCRIPTION
+.B vma_dump_fd_stats()
+Dumps statistics for fd number using log_level log level.
+
```

Comment:
Please consider adding the description of the parameters, for example:
 The argument _fd_ is the file descriptor of the socket, use 0 for all open file descriptors.
 The argument _log_level_ is the dumping level corresponding vlog_levels_t enum (vlogger.h).